pkgs/applications/science/misc/simgrid/default.nix

```diff
@@ -17,14 +17,16 @@ let
 in
 
 stdenv.mkDerivation rec {
-  name = "simgrid-${version}";
-  version = "3.21";
-
-  src = fetchFromGitHub {
-    owner = "simgrid";
-    repo = "simgrid";
-    rev = "v${replaceChars ["."] ["_"] version}";
-    sha256 = "1v0dwlww2wl56ms8lvg5zwffzbmz3sjzpkqc73f714mrc9g02bxs";
+  pname = "simgrid";
+  version = "3.22.2";
+  name = "${pname}-${version}";
```

Comment:
This is not required as it will already be set if `pname` and `version` are supplied.

Reply:
Should be solved by 13aa449bab5, thanks.